### PR TITLE
Receive: fix priority dropdown alignment

### PR DESCRIPTION
**Pull request by selsta**
Also remove unnecessary row layout.

Before:
screenshot 2018-12-28 at 16 36 51

After:
screenshot 2018-12-28 at 16 43 26

Label and dropdown alignment is pixel perfect now :)

### CODE REVIEW

File: pages/Transfer.qml [hunk 1]
```diff
@@ -130,33 +130,27 @@ Rectangle {
 
           ColumnLayout {
               Layout.fillWidth: true
+              Layout.minimumWidth: 200
```
Comment (mmbyday) [inline pages/Transfer.qml]:
```suggestion
              Layout.minimumWidth: 200 * scaleRatio
```

File: pages/Transfer.qml [hunk 2]
```diff
@@ -130,33 +130,27 @@ Rectangle {
...
-                      inlineButtonText: qsTr("All") + translationManager.emptyString
-                      inlineButton.onClicked: amountLine.text = "(all)"
-                      onTextChanged: {
-                          if(amountLine.text.indexOf('.') === 0){
-                              amountLine.text = '0' + amountLine.text;
-                          }
+                  inlineIcon: true
+                  labelText: qsTr("Amount") + translationManager.emptyString
+                  placeholderText: qsTr("") + translationManager.emptyString
+                  width: 100
```
Comment (mmbyday) [inline pages/Transfer.qml]:
```suggestion
                  width: 100 * scaleRatio
```

File: pages/Transfer.qml [hunk 3]
```diff
@@ -165,7 +159,7 @@ Rectangle {
               Layout.fillWidth: true
               Label {
                   id: transactionPriority
-                  Layout.topMargin: 14
+                  Layout.topMargin: 12
```
Comment (mmbyday) [inline pages/Transfer.qml]:
```suggestion
                  Layout.topMargin: 12 * scaleRatio
```

File: pages/Transfer.qml [hunk 4]
```diff
@@ -191,7 +185,7 @@ Rectangle {
               StandardDropdown {
                   Layout.fillWidth: true
                   id: priorityDropdown
-                  Layout.topMargin: 6
+                  Layout.topMargin: 5
```
Comment (mmbyday) [inline pages/Transfer.qml]:
```suggestion
                  Layout.topMargin: 5 * scaleRatio
```

**Comment by selsta** [comment]
Force-pushed, thanks!

**Comment by xmrdsc** [comment]
Thanks!

**Pull request merged**